normalizer/app/streamlit_app.py

```
@@ -1,211 +1,186 @@
-            )
-            column_mapping['number'] = number_col
+            info_placeholder = st.empty()
+            info_placeholder.info("処理中…しばらくお待ちください")
             
-            address_col = st.selectbox(
-                "住所列",
-                options=range(1, len(columns) + 1),
-                format_func=lambda x: f"{x}: {columns[x-1]}",
-                index=2 if len(columns) > 2 else 0,
-                help="住所が含まれる列を選択"
+            results = process_csv_data(
```

Ruff (0.11.9)

`process_csv_data` may be undefined, or defined from star imports

(F405)